### PR TITLE
move s3 redirects to jekyll redirect_from

### PR DESCRIPTION
**Pull request by crazy-max**
follow-up https://github.com/docker/docker.github.io/pull/15109

This is in preparation for a follow-up PR to handle redirects directly in our Cloudfront distribution: https://github.com/crazy-max/docker.github.io/commit/d44d84060943631b7a73d8b9411364edcff8e434

Signed-off-by: CrazyMax <crazy-max@users.noreply.github.com>

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -39,4 +49,24 @@ redirect_from:
...
 - /engine/reference/commandline/cluster_update/
 - /engine/reference/commandline/cluster_version/
+- /v1.4/
+- /v1.5/
+- /v1.6/
+- /v1.7/
+- /v1.8/
+- /v1.9/
+- /v1.10/
+- /v1.10/
```
Comment (thaJeztah) [inline index.md:59]: I'm not sure these would work; the current redirects match by prefix (so `/v1.4/engine/reference/commandline/cli/` (e.g.) would redirect to `/engine/reference/commandline/cli/` etc.

Comment (crazy-max) [reply index.md:59]: Ohhh that's a very good point, didn't catch that we were using `KeyPrefixEquals`

Comment (crazy-max) [reply index.md:59]: I guess we have to handle that with the lambda edge function I was working on.

Comment (thaJeztah) [reply index.md:59]: Yeah, we could add some of those I guess; then again, the current ones works, so perhaps no need to spend too much time on that for now

**Pull request closed without merging**